lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-container-helpers';
+ @use '../../scss-shared/nx-text-helpers';
+ 
+$-box-shadow-offset: 52px;
+
+.nx-drawer {
+  // Undo styles that chrome applies to <dialog> elements
+  border: initial;
+  height: initial;
+  left: initial;
+  margin: initial;
+  max-height: initial;
+  max-width: initial;
+  padding: initial;
+  width: initial;
+
+  // Prevent brief flash of scrollbar as tooltips disappear
+  overflow: hidden;
+
+  &::backdrop {
+    // Chrome tries to add its own subtle mask here which doubles up with our own
+    display: none;
+    pointer-events: auto;
+  }
+
+  background: transparent;
+  box-sizing: border-box;
+  height: 100%;
+  padding-left: $-box-shadow-offset;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: translateX(100%);
+  width: 520px + $-box-shadow-offset;
+  z-index: 100;
+
+  &--with-global-header {
+    $global-header-height: 43px;
+    height: calc(100% - #{$global-header-height});
+    top: $global-header-height;
+  }
+  
+  &--narrow {
+    width: 348px + $-box-shadow-offset;
+  }
+
+  &__inner {
+    align-items: stretch;
+    background-color: var(--nx-swatch-white);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    padding: var(--nx-spacing-6x);
+    position: relative;
+
+    &::after {
+      content: " ";
+
+      box-shadow: -2px 0px 50px rgba(128, 128, 128, 0.5);      
+      height: calc(100% - var(--nx-spacing-16x));
+      width: 100%;      
+      position: absolute;
+      left: 0;
+      top: var(--nx-spacing-8x);
+      z-index: -1;
+    }  
+  }
+
+  transition: transform 120ms ease-in;
+
+  &--open {
+    transform: translateX(0%);
+    transition-duration: 180ms;
+    transition-timing-function: ease-out;
+  }
+
+  @media (prefers-reduced-motion) {
+    &, &--open {
+      transition: unset;
+    }
+  }
+}
+
+.nx-drawer-header {
+  @include nx-container-helpers.container-vertical;
+
+  
+  border-bottom: 1px solid var(--nx-swatch-indigo-90);
+  display: grid;
+  column-gap: var(--nx-spacing-2x);
+  grid-template: "title cancel-button"
+                 "subtitle subtitle"
+                 "paragraph paragraph" / auto min-content;
+  height: min-content;
+  padding-bottom: var(--nx-spacing-6x);
+
+  &__cancel-button, &__title, &__subtitle, &__paragraph {
+    margin: 0;
+  }
+
+  &__cancel-button {
+    grid-area: cancel-button;
+  }
+
+  &__title {
+    @include nx-text-helpers.truncate-ellipsis;
+
+    grid-area: title;
+  }
+
+  &__subtitle {
+    grid-area: subtitle;
+    margin-top: var(--nx-spacing-2x);
+  }
+
+  &__paragraph {
+    color: var(--nx-swatch-grey-30);
```

Comment:
This is our default text color and as such probably doesn't need to be specified

Reply:
> For some reason the paragraph color by default is black.

Oh that's because `<dialog>` has a built-in color.  That should be overridden at the `.nx-drawer` level, not on any specific child element.  We had to do the same thing with `NxModal`.

Reply:
> 

Moved it as part of this commit: 34abea79f4e3011eb4d06cf50ad039a187b341e4